C_CPP/Control-flow-statements.md

```diff
@@ -0,0 +1,28 @@
+# C++ Control Flow Statements
+
+Control flow or flow of control is the order in which instructions, statements and function calls being executed or evaluated when a program is running. The control flow statements are also called as Flow Control Statements. In C++, statements inside your code are generally executed sequentially from top to bottom, in the order that they appear.
+
+## C++ Control Flow Statement Types
+In C++, Control flow statements are mainly categorized in following types –
+
+* Selection statements
```

Comment:
Try to add some more explanation and images for each subtopic and some code examples